### PR TITLE
[incubator/sparkoperator] Upgrade to use the v1beta2-1.0.0-2.4.4 version

### PR DESCRIPTION
**Pull request by liyinan926**
#### What this PR does / why we need it:

Upgrade to use the v1beta2-1.0.0-2.4.4 version of the Spark Operator.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

N/A

#### Special notes for your reviewer:

N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: incubator/sparkoperator/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: sparkoperator
 description: A Helm chart for Spark on Kubernetes operator
-version: 0.3.1
-appVersion: v2.4.0-v1beta1-0.9.0
+version: 0.3.2
```
Comment (yuchaoran2011) [inline incubator/sparkoperator/Chart.yaml]: Let's change the version to 0.4.0 because it's a new release, not a bug fix.

**Review by yuchaoran2011 — COMMENTED** [review]
I'm not sure why the test failed. Could you update the version and sign the DCO? Then let's see if the test will pass.

File: incubator/sparkoperator/Chart.yaml (hunk 1 above)
Comment (liyinan926) [reply incubator/sparkoperator/Chart.yaml]: Done.

**Pull request closed without merging**